Buffs/Overdrive/Overdrive.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using LeagueSandbox.GameServer.Logic.GameObjects;
+using LeagueSandbox.GameServer.Logic.Scripting;
+using LeagueSandbox.GameServer.Logic.API;
+
+namespace Overdrive
+{
+    class Overdrive : BuffGameScript
+    {
+        ChampionStatModifier statMod = new ChampionStatModifier();
+
+        public void OnActivate(Unit unit, Spell ownerSpell)
+        {
```

Comment:
IMO you should initialize it here since you're assuming it's newly initialized but don't actually know if it is or isn't. So move the initialization of `statMod` from line 10 to line 14. This clarifies it a bit and ensures that we're actually always handling a new empty statmod